tecken/libdockerflow.py

```diff
@@ -6,42 +6,31 @@
 from pathlib import Path
 
 from django.core import checks
-from django.conf import settings
 
-from tecken.storage import StorageBucket, StorageError
+from tecken.storage import StorageError
+from tecken.base.symbolstorage import normal_storage
 
 
 def check_storage_urls(app_configs, **kwargs):
     errors = []
-    checked = []
-
-    def check_url(url, setting_key):
-        if url in checked:
-            return
-        bucket = StorageBucket(url)
+    for source in normal_storage().sources:
         try:
-            if not bucket.exists():
+            if not source.exists():
                 errors.append(
                     checks.Error(
-                        f"Unable to connect to {url} (bucket={bucket.name!r}), "
+                        f"Unable to connect to source {source.name!r}, "
```

Comment:
This error is only going to be printed when we haven't created the storage bucket yet. In practice this will probably only happen when we messed up the local development environment somehow. I don't really know why I removed the URL – it's available as `source.url`. I'll add it back.